src/status_im2/contexts/chat/messages/content/text/style.cljs

```diff
@@ -20,6 +23,30 @@
    :padding-left      10
    :border-left-color colors/neutral-40})
 
+(def mention-tag-wrapper
+  {:position           :relative
+   :padding-vertical   0
+   :padding-horizontal 0
+   :margin-vertical    0
+   :margin-horizontal  0
+   :flex-direction     :row
+   :align-items        :center})
+
+(def mention-tag
+  {:background-color    colors/primary-50-opa-10
+   :position            :relative
+   :padding-horizontal  3
+   :border-radius       6
+   :margin-bottom       -3
+   :z-index             1})
+
+(def mention-tag-text
+  {:color                 (colors/theme-colors colors/primary-50
+                                               colors/primary-60)
+   :selection-color       :transparent
+   :include-font-padding  false
```

Comment:
Should we add `:text-align-vertical :center` or it doesn't make much of a difference?

> For best results also set textAlignVertical to center.
>
> -- https://reactnative.dev/docs/text-style-props#includefontpadding-android